pages/pipelines/artifacts.md.erb

```diff
@@ -6,7 +6,25 @@ In this guide we’ll walk through using the Buildkite Agent’s [artifact suppo
 
 ## Uploading Artifacts
 
-To automatically upload artifacts created by a build step, set the `artifact_paths` attribute on your command step. In this example, any artifacts that are created in the `logs` or `coverage` directories, as well as any of their subdirectories, will be uploaded automatically:
+How you define your build pipeline determines how you automatically upload artifacts created by build steps:
+
+* pipelines defined in the Buildkite UI use the ["Automatic Artifact Uploading"](#uploading-artifacts-automatic-artifact-uploading) option.
```

Comment:
Could we pull out 'Pipelines defined in' to the top of the list, then have each dot point start from there? Also, we tend to not do full stops at the end of dot points unless there're multiple sentences 😊

Reply:
Thanks for patiently walking me through the style guide. 

I tried pulling out the "Defined in", but i don't really like it. Both sentences end up super fragmented. Let's chat about the broader layout tomorrow.